gremlin/ui/device.py

```diff
@@ -194,6 +290,7 @@ def __init__(self, parent=None):
         super().__init__(parent)
 
         self._device: dill.DeviceSummary | None = None
+        self.device_mapping: Dict[str, str] | None = None
```

Comment:
Since we don't expect to expose this I'd turn it into a private variable, i.e. `self._device_mapping`

Reply:
ok